charts/spire/charts/spire-server/templates/hpa.yaml

```diff
@@ -12,6 +12,9 @@ spec:
     kind: Deployment
     name: {{ include "spire-server.fullname" . }}
   minReplicas: {{ .Values.autoscaling.minReplicas }}
+  {{- if gt (int .Values.autoscaling.maxReplicas) 1 }}
```

Comment:
If external db merges first, then this becomes something like:
{{- if and (eq .Values.dataStore.sql.databaseType "sqlite3") (gt (int .Values.autoscaling.maxReplicas) 1) }}

Reply:
Yes, lets merge the external db first and then this change.

Reply:
Reordered to come after external db change.